bitcoin/src/crypto/key.rs

```diff
@@ -904,8 +904,10 @@ mod tests {
             k.write_into(&mut v).expect("writing into vec");
         }
 
+        // Note that we must temporarily use std::io::Cursor here until libsecp256k1 converts to
+        // bitcoin_io.
```

Comment:
Are you planning on doing this yourself after this merges? If not can you please use the form `// TODO: ...` so I find it when I'm clearing todo's.

Reply:
Where's `secp256k1` using IO? `rg [^a-z]io[^a-z]` doesn't return anything relevant neither does `rg core2`

Also `Cursor` wasn't required here in the first place, just use `&mut &[u8]`.